mobile/src/components/course/components/edit-course-category/common/types/dropdown.type.ts

```diff
@@ -0,0 +1,5 @@
+type DropDownPayload = {
```

Comment:
Remove this, why don't we use something like CourseUpdateCategoryRequestDto that should already exist?

Reply:
Then use empty object for default values